<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserConfig</class>
 <widget class="QDialog" name="UserConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>436</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>525</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configurações do usuário</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QPushButton" name="pushButtonAlterarDados">
     <property name="text">
      <string>Alterar dados do usuário</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButtonSalvar">
     <property name="text">
      <string>Salvar</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">QTabBar::tab:disabled {
	 width: 0;
	 height: 0;
	 margin: 0;
	 padding: 0;
	 border: none; 
}

QTabWidget::pane {
	margin: 0px 0px 0px 0px; 
}
</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabArquivos">
      <attribute name="title">
       <string>Arquivos</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Pasta PDF/Excel</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditOrcamentosFolder">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="pushButtonVendasFolder">
            <property name="text">
             <string>Navegar...</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Vendas:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Orçamentos:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButtonOrcamentosFolder">
            <property name="text">
             <string>Navegar...</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditVendasFolder">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelCompras">
            <property name="text">
             <string>Compras:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEditComprasFolder">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="pushButtonComprasFolder">
            <property name="text">
             <string>Navegar...</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelEntregas">
            <property name="text">
             <string>Entregas XML:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEditEntregasXmlFolder">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QPushButton" name="pushButtonEntregasXmlFolder">
            <property name="text">
             <string>Navegar...</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelEntregas_2">
            <property name="text">
             <string>Entregas PDF:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEditEntregasPdfFolder">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QPushButton" name="pushButtonEntregasPdfFolder">
            <property name="text">
             <string>Navegar...</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabEmail">
      <attribute name="title">
       <string>Email</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxEmail">
         <property name="title">
          <string>E-mail</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="10" column="1" colspan="2">
           <widget class="QPushButton" name="pushButtonEmailTeste">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Enviar email de teste</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEditPortaSMTP">
            <property name="placeholderText">
             <string>465</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Senha:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Assinatura:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEditEmailCopia">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButtonSelecionarAssinatura">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Selecionar...</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Servidor SMTP:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEditEmailSenha">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditAssinaturaEmail">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEditEmail">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>E-mail:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Cópia p/:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEditServidorSMTP">
            <property name="placeholderText">
             <string>email-ssl.com.br</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Porta:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QTextBrowser" name="textBrowserMensagemEmail">
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Mensagem&lt;/p&gt;&lt;p&gt;padrão:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabNFe">
      <attribute name="title">
       <string>NF-e</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxAcbr">
         <property name="title">
          <string>Emissor de NF-e - ACBr</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="3" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Emitente:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Porta:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>E-mail Contabilidade:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="3">
           <widget class="QLineEdit" name="lineEditEmailContabilidade">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditACBrServidor">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="3">
           <widget class="ItemBox" name="itemBoxLoja"/>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="lineEditACBrPorta">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Servidor:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="3">
           <widget class="QLineEdit" name="lineEditEmailLogistica">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>E-mail Logística:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBoxDownloadNFe">
         <property name="title">
          <string>Download automático de NF-es</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <property name="spacing">
              <number>3</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="labelRaizCNPJ">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Empresa 1:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="3">
              <widget class="QComboBox" name="comboBoxMonitorar1"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Servidor:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEditMonitorarServidor1">
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Porta:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="lineEditMonitorarPorta1">
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <property name="spacing">
              <number>3</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="labelRaizCNPJ_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Empresa 2:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="3">
              <widget class="QComboBox" name="comboBoxMonitorar2"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Servidor:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEditMonitorarServidor2">
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Porta:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="lineEditMonitorarPorta2">
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemBox</class>
   <extends>QLineEdit</extends>
   <header>src/itembox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditACBrServidor</tabstop>
  <tabstop>lineEditACBrPorta</tabstop>
  <tabstop>itemBoxLoja</tabstop>
  <tabstop>lineEditEmailContabilidade</tabstop>
  <tabstop>lineEditEmailLogistica</tabstop>
  <tabstop>lineEditServidorSMTP</tabstop>
  <tabstop>lineEditPortaSMTP</tabstop>
  <tabstop>lineEditEmail</tabstop>
  <tabstop>lineEditEmailSenha</tabstop>
  <tabstop>lineEditEmailCopia</tabstop>
  <tabstop>pushButtonEmailTeste</tabstop>
  <tabstop>lineEditOrcamentosFolder</tabstop>
  <tabstop>pushButtonOrcamentosFolder</tabstop>
  <tabstop>lineEditVendasFolder</tabstop>
  <tabstop>pushButtonVendasFolder</tabstop>
  <tabstop>lineEditComprasFolder</tabstop>
  <tabstop>pushButtonComprasFolder</tabstop>
  <tabstop>lineEditEntregasXmlFolder</tabstop>
  <tabstop>pushButtonEntregasXmlFolder</tabstop>
  <tabstop>lineEditEntregasPdfFolder</tabstop>
  <tabstop>pushButtonEntregasPdfFolder</tabstop>
  <tabstop>pushButtonAlterarDados</tabstop>
  <tabstop>pushButtonSalvar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
